src/main/java/org/miracum/etl/fhirtoomop/listeners/AppointmentStepListener.java

```diff
@@ -0,0 +1,80 @@
+package org.miracum.etl.fhirtoomop.listeners;
+
+import com.google.common.base.Strings;
+import lombok.extern.slf4j.Slf4j;
+import org.miracum.etl.fhirtoomop.DbMappings;
+import org.miracum.etl.fhirtoomop.MemoryLogger;
+import org.miracum.etl.fhirtoomop.repository.OmopRepository;
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.StepExecution;
+import org.springframework.batch.core.StepExecutionListener;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StopWatch;
+import javax.sql.DataSource;
+
+import static org.miracum.etl.fhirtoomop.Constants.*;
```

Comment:
Please avoid * import

Reply:
These import is not being used anywhere. Removing it.